ddtrace/tracer/tracer.go

```diff
@@ -6,6 +6,7 @@
 package tracer
 
 import (
+	"context"
```

Comment:
nit: it looks like we already have this imported as `gocontext` (although it's not clear to me why we bothered to give it a different name). Can we drop one of these?

Reply:
I changed the PR to use `gocontext`. Changing all the uses of `gocontext` to `context` caused some compilation errors that didn't seem worth fixing (other uses of the name `context`).